### ISSUE TITLE
Output of Postgres `generated_user_password` IS Fixed on Random Password 

### ISSUE THREAD

**Issue by moshem-pangea** [terraform-google-modules/terraform-google-sql-db#379]
### TL;DR

The output `generated_user_password` seems to always return the auto-generated random password, even in cases when the user sets own password.

### Expected behavior

It should return the value of [google_sql_user.default.password](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/f36347eb049389c89ba019100668ba481702df90/modules/postgresql/main.tf#L197)

### Observed behavior

Instead it always returns the random password - see [here](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/f36347eb049389c89ba019100668ba481702df90/modules/postgresql/outputs.tf#L96)

### Terraform Configuration

```hcl
output "psql_user_pass" {
  value       = module.postgresql-db.generated_user_password
  description = "The password for the default user. If not set, a random one will be generated and available in the generated_user_password output variable."
  sensitive   = true
}
```

### Terraform Version

```sh
1.3.5
```
